pages/myProfile/profilePage.module.scss

```diff
@@ -0,0 +1,6 @@
+.userPageRoot {
```

Comment:
The class name is `.userPageRoot`, but the associated component is called `profilePage`. This is incompatible. Prefer matching names to avoid confusion among consumers of your component.